polkadot-protocol/protocol-components/accounts/account-data-structure.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Account Data Structure
+description: TODO
+---
+
+# Account Data Structure
+
+!!! warning "Update Notice"
+    Please read Substrate to [Polkadot SDK](https://docs.substrate.io/polkadot-sdk/) page first.
+
+Accounts are a foundational element of any blockchain, and in Substrate, they serve various functions. This article explains how accounts are stored in Substrate and how their data structures are utilized to manage the account lifecycle within the runtime logic.
+
+## Account
+
+The `Account` data type is a generically defined storage map within the [`frame-system`](https://paritytech.github.io/polkadot-sdk/master/src/frame_system/lib.rs.html) pallet:
+
+```rs
```

Comment:
Not sure it is productive to have this snippet here, in the comment above I linked the struct itself in the codebase, which is more maintainable IMO.

Instead, a simple explanation of how it is a mapping of `AccountId` -> `AccountInfo` would suffice

Reply:
I agree